src/include/cart/iv.h

```diff
@@ -394,62 +394,32 @@ struct crt_iv_class {
  *				See \a enum crt_tree_type,
  *				\a crt_tree_topo().
  * \param[in] iv_classes	the array of IV class. User must ensure passing
- *				same set of iv_classes when creating or
- *				attaching IV namespace, or undefined results are
- *				expected.
+ *				same set of iv_classes when adding IV namespace
+ *				on all participating nodes.
  * \param[in] num_class		the number of elements in the iv_classes array,
  *				one IV namespace should have at least one IV
  *				class.
+ * \param[in] iv_ns_id		Unique id, identifying the namespace within the
+ *				group.
  * \param[out] ivns		the local handle of the IV namespace
- * \param[out] g_ivns		the global handle of the IV namespace. It can be
- *				transferred to other processes on any other node
- *				in the group, and the crt_iv_namespace_attach()
- *				can attach to the global handle to get a local
- *				usable IV namespace handle.
  *
  * \return			DER_SUCCESS on success, negative value if error
  */
 int
-crt_iv_namespace_create(crt_context_t crt_ctx, crt_group_t *grp, int tree_topo,
-			struct crt_iv_class *iv_classes, uint32_t num_class,
-			crt_iv_namespace_t *ivns, d_iov_t *g_ivns);
+crt_iv_namespace_add(crt_context_t crt_ctx, crt_group_t *grp, int tree_topo,
```

Comment:
Why renaming "create" to "add"? For RPM/build ABI reasons? I think "create" is good.